tools/topology/topology2/include/pipelines/cavs/mixout-gain-efx-alh-dai-copier-playback.conf

```diff
@@ -0,0 +1,199 @@
+#
```

Comment:
@singalsu lets update this to use the subtreecopy construct like we discussed

Reply:
@ranj063 Done now! I hope I've understood it right, does this look OK to you?

I also added to last commit enable of processing for two topologies those are tested by SOF CI to see they are working fully. Earlier changed topologies did not get tested except for the "false" path of tplg changes.